server/controllers/userAuth.js

```diff
@@ -163,7 +163,7 @@ const resendSignUpConfirmationLink = (req, resp, next) => {
     },
     {
       signUpHash: crypto.randomBytes(32).toString('hex'),
-      signUpHashExpirationDate: new Date().getTime() + 3600000
+      signUpHashExpirationDate: new Date().getTime() + EXPIRATION_TIME
```

Comment:
Do the sign-up and other emails now contain the expiration message as well?

Reply:
Two messages contain this message right now.
When signing up and when resetting the password.

Reply:
Ok